polkadot/zombienet_tests/functional/0007-approval-voting-coalescing.toml

```diff
@@ -0,0 +1,195 @@
+[settings]
+timeout = 1000
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+chain_spec_command = "polkadot build-spec --chain rococo-local --disable-default-bootnode"
+
+[relaychain.genesis.runtime.configuration.config]
+  needed_approvals = 5
+  relay_vrf_modulo_samples = 6
+
+[relaychain.genesis.runtime.configuration.config.approval_voting_params]
+  max_approval_coalesce_count = 5
+
+[relaychain.default_resources]
+limits = { memory = "4G", cpu = "2" }
+requests = { memory = "2G", cpu = "1" }
+
+  [[relaychain.nodes]]
+  name = "alice"
+  args = [ "--alice", "-lparachain=trace,runtime=debug" ]
```

Comment:
can we use node groups here?
https://github.com/paritytech/zombienet/blob/d8ded0cb017b742e91298a70d0654025deedf505/examples/0003-big-network.toml#L8-L11

Reply:
Will do.